src/components/autosuggest/_macro.njk

```diff
@@ -36,14 +36,13 @@
     >
 
         {% set autosuggestResults %}
-            <div class="ons-autosuggest__results ons-js-autosuggest-results">
+            <div class="ons-autosuggest__results ons-js-autosuggest-results {% if params.width %} ons-input{% if params.type == 'number' or params.type == 'tel' %}-number{% endif %}--w-{{ params.width }}{% endif %} {% if not params.width %} ons-input--w-20 {% endif %}">
```

Comment:
```suggestion
            <div class="ons-autosuggest__results ons-js-autosuggest-results{% if params.width %} ons-input{% if params.type == 'number' or params.type == 'tel' %}-number{% endif %}--w-{{ params.width }}{% else %} ons-input--w-20{% endif %}">
```
I would probably do this, makes it a bit simpler and removes some extra spaces from the html.

But I'm also not sure we need the `params.type` stuff. Should always be text which I think is why `params.type` isn't currently passed in.